src/axe-report.ts

```diff
@@ -38,14 +38,33 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
     });
   });
 */
+import * as axeTypes from 'axe-core';
 
-export async function axeReport(dom, config = {}) {
+declare global {
+  namespace axe {
+    const run: typeof axeTypes.run;
+  }
+}
+
+export interface AxeReportOptions extends axeTypes.RunOptions {
+  cleanup?: () => Promise<void>
+  axeConfig?: axeTypes.RunOptions
+}
+
+interface AxeReportRunOptions extends axeTypes.RunOptions {
+  resultTypes: string[]
```

Comment:
This is missing from the upstream type? Consider submitting a PR with this and the other missing pieces to DefinitelyTyped

Reply:
this is correct. My PR got merged today looks like it has not been released:

https://github.com/dequelabs/axe-core/pull/568